include/picongpu/plugins/binning/utility.hpp

```diff
@@ -107,6 +107,22 @@ namespace picongpu
             return std::unique_ptr<std::remove_cvref_t<decltype(*ptr)>>(ptr);
         }
 
+        template<typename, template<typename...> typename>
+        struct is_specialization_of : std::false_type
```

Comment:
Discussed offline that this will go into pmacc and should follow our naming rules.